SU2_CFD/src/fluid/CFluidScalar.cpp

```diff
@@ -0,0 +1,80 @@
+/*!
+ * \file CFluidScalar.hpp
+ * \brief Defines the incompressible Ideal Gas model.
+ * \author T. Economon, Mark Heimgartner, Cristopher Morales Ubal 
+ * \version 7.3.1 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <cmath>
+#include <math.h>
+#include <vector>
+#include <numeric>
+#include <iostream>
+
+#include "../../include/fluid/CFluidScalar.hpp"
+#include "../../include/fluid/CConstantViscosity.hpp"
+#include "../../include/fluid/CSutherland.hpp"
+#include "../../include/fluid/CPolynomialViscosity.hpp"
+#include "../../include/fluid/CConstantConductivity.hpp"
+#include "../../include/fluid/CConstantConductivityRANS.hpp"
+#include "../../include/fluid/CConstantPrandtl.hpp"
+#include "../../include/fluid/CConstantPrandtlRANS.hpp"
+#include "../../include/fluid/CPolynomialConductivity.hpp"
+#include "../../include/fluid/CPolynomialConductivityRANS.hpp"
+#include "../../include/fluid/CIncIdealGas.hpp"
```

Comment:
Remove all includes that you don't need.

Reply:
done, Thank you so much for the comments!!!